services/auth/test/stories_test.js

```diff
@@ -160,8 +161,7 @@ helper.secrets.mockSuite(testing.suiteName(), ['azure', 'gcp'], function(mock, s
       });
     });
 
-    // TODO: bug 1242473
```

Comment:
I wonder if an enterprising person could cook up a test that checks the status of `TODO: bug XXXX` type comments and throw if they are closed or something. It would only run on master or such.

Reply:
If we were going to be enterprising, I'd kind of prefer to just get rid of them, and add a meta-check that no further TODO's are added.  I'm not terribly motivated beyond the effort that went into this PR, though.